scripts/invoke_update_shard.sh

```
@@ -1,23 +1,49 @@
+# It takes one argument, which is the operation to perform.
+# The operations are:
+# - add: sums the diff value of the slot with the previous value
+# - lock_set: locks the slot and sets a new value for it
+# - set: sets a new value for the slot
+# - lock: locks the contract slots (locked slots are immutable)
+# It's used only for one slot "counter" in the contract.
+# Only one argument is needed, the operation to perform (add, lock_set, set, lock) 
+# and it will test these scenarios.
+
 SHARDING_CONTRACT_ADDRESS=$(cat sharding_contract_address.txt)
 TEST_CONTRACT_ADDRESS=$(cat test_contract_address.txt)
```

Shellcheck (0.10.0)

[warning] TEST_CONTRACT_ADDRESS appears unused. Verify use (or export if used externally).

(SC2034)